### PR TITLE
docs: fix k9s plugin file description

### PR DESCRIPTION
**Pull request by sullrich84**

### CODE REVIEW

**Comment by bomgar** [comment]
Did they change this? I just tried and I need to name the file `plugins.yaml` instead of `plugins.yml`. Also Shift-A seems to be no longer bindable.
Could you check that it is `plugins.yaml` for you, too?

**Comment by sullrich84** [comment]
This is what I figured out yesterday. `k9s` info gives me:

```
 ____  __.________       
|    |/ _/   __   \______
|      < \____    /  ___/
|    |  \   /    /\___ \ 
|____|__ \ /____//____  >
        \/            \/ 

Version:           0.32.7
Config:            /Users/ullrichs/Library/Application Support/k9s/config.yaml
Custom Views:      /Users/ullrichs/Library/Application Support/k9s/views.yaml
Plugins:           /Users/ullrichs/Library/Application Support/k9s/plugins.yaml
Hotkeys:           /Users/ullrichs/Library/Application Support/k9s/hotkeys.yaml
Aliases:           /Users/ullrichs/Library/Application Support/k9s/aliases.yaml
Skins:             /Users/ullrichs/Library/Application Support/k9s/skins
Context Configs:   /Users/ullrichs/Library/Application Support/k9s/clusters
Logs:              /Users/ullrichs/Library/Application Support/k9s/k9s.log
Benchmarks:        /Users/ullrichs/Library/Application Support/k9s/benchmarks
ScreenDumps:       /Users/ullrichs/Library/Application Support/k9s/screen-dumps
```

and the official documentation also suggests `plugins` as a file name and root node: https://k9scli.io/topics/plugins/

File: README.md [hunk 1]
```diff
@@ -193,10 +193,10 @@ You can map additional level values (used for output and color):
 
 ## k9s
 
-In the file `~/.config/k9s/plugin.yml` add this:
+In the file `~/.config/k9s/plugins.yml` add this:
```
Comment (bomgar) [inline README.md:196]:
```suggestion
In the file `~/.config/k9s/plugins.yaml` add this:
```

**Comment by bomgar** [comment]
Thank you

**Pull request merged**